test/tests/functional/pbs_svr_dyn_res.py

```diff
@@ -596,7 +596,7 @@ def test_svr_dyn_res_permissions(self):
         self.scheduler.add_resource('foo')
 
         scr_body = ['echo "10"', 'exit 0']
-        home_dir = os.path.expanduser("~")
+        home_dir = os.path.join('/', 'tmp')
```

Comment:
why /tmp?  Also use os.sep, not '/'

Reply:
if the daemon user doesn't have read permissions to the current user, it won't be able to read the file.

Reply:
Moving this to /tmp will cause chk_file_sec() to fail in a different way.  It will still not allow the script, but it will fail because /tmp has rwx for all perms.  The doc string wants it to fail based on the fact that the script itself has the wrong perms.